### PR TITLE
fix: new elevation service api

### PR DESCRIPTION
**Pull request by alebinson**

### CODE REVIEW

File: static/openapi/dem/elevation-openapi.yaml [hunk 1]
```diff
@@ -106,18 +100,31 @@ components:
                 nullable: true
                 type: number
                 format: double
+              productId:
```
Comment (shimoncohen) [inline static/openapi/dem/elevation-openapi.yaml:103]: Can you please change the example to show the correlation between the data productId to one of the returned products?

File: docs/MapColonies/DEM/Services/elevation/README.md [hunk 2]
```diff
@@ -62,15 +59,26 @@ Limited up to **250** coordinates
...
+  "products": {
+    "world_terrain": {
+      "productType": "DTM",
+      "resolutionMeter": 30,
+      "absoluteAccuracyLEP90": 17,
+      "updateDate": "2019-08-24T14:15:22Z"
+    }
+  }
+}
+```
```
Comment (shimoncohen) [inline docs/MapColonies/DEM/Services/elevation/README.md:84]: Could you please explain how users could know the product's geometry? So they could know the product's boundaries.

**Review by shimoncohen — CHANGES_REQUESTED** [review]

File: static/openapi/dem/elevation-openapi.yaml (hunk 1 above)
Comment (alebinson) [reply static/openapi/dem/elevation-openapi.yaml:103]: Done, examples are provided

File: docs/MapColonies/DEM/Services/elevation/README.md (hunk 2 above)
Comment (alebinson) [reply docs/MapColonies/DEM/Services/elevation/README.md:84]: No need to explain about geometries

**Review by shimoncohen — APPROVED** [review]

**Pull request merged**